wait/host_port.go

```diff
@@ -57,8 +57,23 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 
 	var waitInterval = 100 * time.Millisecond
 
+	internalPort := hp.Port
+	if internalPort == "" {
```

Comment:
Can we explicitly mention that if the port is the empty string then the first internal exposed port will be used?